Bottom: 1ce657e21ebec0145323ef55078905d41743cc63
Top:    8664b83ab73eb27e20844b8a055699ea71740d20
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-03-27 00:08:23 +0000

Refresh of hid-gtk-add-hooks-for-the-rend

---

diff --git a/src/hid/gtk/gui.h b/src/hid/gtk/gui.h
index eb1e853..69bd992 100644
--- a/src/hid/gtk/gui.h
+++ b/src/hid/gtk/gui.h
@@ -496,6 +496,7 @@ void ghid_invalidate_lr (int left, int right, int top, int bottom);
 void ghid_invalidate_all ();
 void ghid_show_crosshair (gboolean show);
 void ghid_init_renderer (int *, char ***, GHidPort *);
+void ghid_init_drawing_widget (GtkWidget *widget, GHidPort *);
 void ghid_drawing_area_configure_hook (GHidPort *port);
 void ghid_screen_update (void);
 gboolean ghid_drawing_area_expose_cb (GtkWidget *, GdkEventExpose *,
